Modules/posixmodule.c

```diff
@@ -3535,16 +3568,26 @@ os_lchmod_impl(PyObject *module, path_t *path, int mode)
     if (PySys_Audit("os.chmod", "Oii", path->object, mode, -1) < 0) {
         return NULL;
     }
+#ifdef MS_WINDOWS
+    Py_BEGIN_ALLOW_THREADS
+    res = win32_lchmod(path->wide, mode);
+    Py_END_ALLOW_THREADS
+    if (!res) {
+        path_error(path);
```

Comment:
```suggestion
        (void)path_error(path);
```

Reply:
I don't think sloppiness elsewhere in the code is a good reason not to do it in new code. IMO, we should eventually fix the other occurrences.

Reply:
No. There is nothing wrong in ignoring the result of these functions. They always return NULL, but this is for convenience, ignoring it is not error.

Otherwise you will need to change every call of `PyErr_Format()` and other non-void functions.

Reply:
Fair enough.

Reply:
A neater (and IMO better) variant would be to consolidate this line and the following `return NULL` to `return path_error(path)`.

Reply:
Yes, it is why `path_error()` returns NULL at first place. But the old code did not use it, so I did not change it.